packages/shopping/public/reset-password-finish.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+    Shoppy: The #1 online shopping destination
+    </title>
+    <link rel="icon" type="image/png" href="/favicon.png" />
+    <link href="https://fonts.googleapis.com/css?family=Pacifico&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  </head>
+  <body>
+    <div id="main" class="container-fluid">
+      <nav id="navBar" class="navbar navbar-expand-lg navbar-light bg-light"></nav>
+      <div class="card" data-cy="resetPasswordFinish" id="resetPasswordFinish"></div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+    <script type="text/javascript" src="/purify.js"></script>
+    <script type="text/javascript" src="/templates.js"></script>
+    <script type="text/javascript" src="/config.js"></script>
+    <script type="text/javascript" src="/api.js"></script>
+    <script type="text/javascript" src="/util.js"></script>
+    <script type="text/javascript" src="/ui.js"></script>
+    <script type="text/javascript" src="/reset-password-finish.js"></script>
```

Comment:
Some remarks:

1. Add sub-resource integrity for js query
2. In accordance with WHATWG Living HTML Spec, `type` is not mandatory and should be removed as it's redundant. Also, `text/javascript` is deprecated in favour of `application/javascript`

```suggestion
    <script src="https://code.jquery.com/jquery-3.4.1.min.js" integrity="sha384-vk5WoKIaW/vJyUAd9n/wmopsmNhiy+L2Z+SBxGYnUkunIxVxAv/UtMOhba/xskxh" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
    <script src="/purify.js"></script>
    <script src="/templates.js"></script>
    <script src="/config.js"></script>
    <script src="/api.js"></script>
    <script src="/util.js"></script>
    <script src="/ui.js"></script>
    <script src="/reset-password-finish.js"></script>
```